tests/swoole_redis_coro/stream.phpt

```diff
@@ -0,0 +1,124 @@
+--TEST--
+swoole_redis_coro: stream
+--SKIPIF--
+<?php require __DIR__ . '/../include/skipif.inc'; ?>
+--FILE--
+<?php
+require __DIR__ . '/../include/bootstrap.php';
+
+//Co::set(['log_level' => SWOOLE_LOG_TRACE, 'trace_flags' => SWOOLE_TRACE_ALL]);
+
+go(function () {
```

Comment:
Use `Co\run()` instead of `go() + event::wait()`

Reply:
thanks for your help!

Reply:
What else should I modify? @twose